Automattic/wp-calypso
# Checkoutv2: Implement checkout redesign ab test

This PR adds the `useCheckoutV2()` hook to return a string based on the state the checkout experiment is in - either 'loading', 'treatment', or 'control'.

Additionally, this code allows you to see the checkout v2 treatment without the feature flag. You can assign yourself to the treatment by following these steps: PCYsg-SwK-p2 (scroll down to 'Manually assign with Abacus assignment bookmarklet').

Related to pbOQVh-45k-p2 

## Testing Instructions

* Go to Checkout and ensure the v1 version looks correct - look for any style or functionality bleed
* Assign yourself to the treatment group according to the instructions in the link above
* Go to Checkout again and make sure the v2 styles and functionality match those found under the checkout v2 feature flag
* Test a number of products and use cases to ensure that styling remains the same between the feature flagged v2 and the non-flagged 'treatment' v2
